package.json

```diff
@@ -62,6 +61,7 @@
     "ember-cli-uglify": "^3.0.0",
     "ember-cli-yuidoc": "0.9.1",
     "ember-code-snippet": "^3.0.0",
+    "ember-composable-helpers": "^3.1.1",
```

Comment:
Should we just remove this as well?

Reply:
Probably but until I've finished digging through the dummy app I can't be sure it's not in use somewhere...
I've got a WIP PR for the dummy app where I can clean this up if not needed.

Reply:
Ah ok.  If it was removed and the PR passed, then we could do without?

Reply:
@snewcomer I've checked and we use pipe quite a bit in the dummy app so it will need to stay for the timebeing.